src/layouts/MainLayout/TopBar/Account.js

```diff
@@ -16,9 +15,8 @@ import {
   CircularProgress,
   makeStyles
 } from '@material-ui/core';
-import authService from 'src/services/authService';
-import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
 import { login, dismissLogin, logout } from 'src/actions/accountActions';
+import Login from 'src/components/Login';
 
 const useStyles = makeStyles(theme => ({
   button: {
```

Comment:
To fixup the border-radius thing:

add one more class:
```
  dialog: {
    '& .MuiDialog-paper': {
      borderRadius: '20px'
    }
  }
```

and put that class here :
```diff
        <Dialog
          maxWidth
          open={!user && !!loginFlag}
          onClose={handleClose}
          aria-labelledby="form-dialog-title"
+          className={classes.dialog}
        >
          <Login handleClose={handleClose} />
        </Dialog>
```